kube/resources/scheduler.yaml

```diff
@@ -204,20 +204,6 @@ spec:
             - name: gcs-log-creds-volume
               mountPath: /secrets/gcs-log-creds
               readOnly: true
-      initContainers:
-        - name: airbyte-seed
-          image: airbyte/seed
-          command: ["/bin/sh"]
-          args:
-            [
-              "-c",
-              "mkdir -p /configs/config && yes n | cp -r -i /app/seed/config /configs",
-            ]
-          volumeMounts:
-            - name: airbyte-volume-configs
-              mountPath: /configs
```

Comment:
The `data` volume is not removed yet, because users running old Airbyte versions still have configs there. Is this the `data` volume in the kube world?

Reply:
Yeap. In docker the underlying volume was `data` [mounted at](https://github.com/airbytehq/airbyte/pull/5335/files#diff-3fde9d1a396e140fefc7676e1bd237d67b6864552b6f45af1ebcc27bcd0bb6e9L37) `/app/seed` the files were then [copied into](https://github.com/airbytehq/airbyte/pull/5335/files#diff-3fde9d1a396e140fefc7676e1bd237d67b6864552b6f45af1ebcc27bcd0bb6e9L37) `/app/seed/config`.

The equivalent here is the `/configs` volume and we (previously) copied into `/configs/config`.